flatpak/org.nanuc.Axolotl.yml

```diff
@@ -52,11 +52,9 @@ modules:
     buildsystem: simple
     build-options:
       # Add the node bin directory.
-      append-path: "/usr/lib/sdk/node18/bin:/run/build/axolotl/flatpak-node/vite:/usr/lib/sdk/rust-stable/bin"
```

Comment:
I think vite is used by `yarn run build`.

Reply:
Thanks for the review!
The path adjustment is needed for yarn and cargo - but we dont need vite. Vite is available as a npm/yarn script, and that is where it is used too. The change is to remove vite.

Reply:
Can you elaborate why this change is needed for yarn and cargo? They already have been in path before.

Reply:
Returning to this - could you show an example where vite is used outside of package.json?

Reply:
It's only used in package.json but with nopm